pkg/operator/ceph/cluster/mon/spec.go

```diff
@@ -83,6 +84,56 @@ func (c *Cluster) makeDeployment(monConfig *monConfig, hostname string) *apps.De
 	return d
 }
 
+func makeDeploymentPVCName(m *monConfig) string {
+	return m.ResourceName + "-pv-claim"
```

Comment:
How about not using a suffix? Since the type is PVC, using the `m.ResourceName` seems sufficient. You'd end up with a deployment and a PVC with the same name, which seems natural.

Reply:
i have no objections to that at all. reviewing some other code bases with pvc stuff i saw both patterns. relevant tweet https://twitter.com/dotnwat/status/1155940432351641601